src/test/java/uk/gov/ons/ctp/response/sample/service/impl/SampleServiceImplTest.java

```diff
@@ -112,16 +120,13 @@ public void setUp() throws Exception {
    * @throws Exception oops
```

Comment:
If none of these throw Exception any more please update Javadoc (bonus: it gets rid of "oops" :) )

Reply:
Done